### PR TITLE
HDFS assetstore was breaking assetstore page

### PR DESCRIPTION
**Pull request by zachmullen**
when server was not reachable.

### CODE REVIEW

File: plugins/hdfs_assetstore/server/assetstore.py [hunk 1]
```diff
@@ -101,11 +101,17 @@ def validateInfo(doc):
...
-            'free': info['capacity'] - info['used'],
-            'total': info['capacity']
-        }
+        try:
+            info = self.client.df()
+            return {
+                'free': info['capacity'] - info['used'],
+                'total': info['capacity']
+            }
+        except:
```
Comment (manthey) [inline plugins/hdfs_assetstore/server/assetstore.py]: Can you change this to a more specific exception?  Even just `except Exception:` would be better (so it doesn't catch base exceptions).

Comment (manthey) [inline plugins/hdfs_assetstore/server/assetstore.py]: I've added a PR with a flake8 test for this condition.

**Comment by zachmullen** [comment]
ping @manthey

**Comment by manthey** [comment]
LGTM.

**Pull request merged**